example/passthrough_hp.cc

```diff
@@ -259,9 +278,9 @@ static void do_setattr(fuse_req_t req, fuse_ino_t ino, struct stat *attr,
 		if (fi) {
 			res = fchmod(fi->fh, attr->st_mode);
 		} else {
-			char procname[64];
-			sprintf(procname, "/proc/self/fd/%i", ifd);
-			res = chmod(procname, attr->st_mode);
+			res = with_fd_path(ifd, [attr](const char* procname) {
+				return chmod(procname, attr->st_mode);
```

Comment:
Yeah, I wonder if we could just remove the O_PATH in do_lookup() and main and than use inode.fd. A bit sad to have an fd and then not to be able to use it...